services/spar/src/Spar/Data.hs

```diff
@@ -478,37 +478,37 @@ deleteTeamScimTokens team = do
 ----------------------------------------------------------------------
 -- SCIM user records
 
--- | Store the scim user in its entirety and return the 'SCIM.StoredUser'.
+-- | Store the scim user in its entirety and return the 'Scim.StoredUser'.
 --
 -- NB: we can add optional columns in the future and extract parts of the json blob should the need
 -- arise.  For instance, if we want to support different versions of SCIM, we could extract
--- 'SCIM.User.schemas' and, throw an exception if the list of values is not supported, and store it
```

Comment:
`SCIM.User.schemas` is no longer a part of `User`, though it's still present in JSON.